pyface/tests/test_image_resource.py

```diff
@@ -28,9 +29,9 @@
 )
 
 
-SEARCH_PATH = pkg_resources.resource_filename("pyface", "images")
-IMAGE_DIR = pkg_resources.resource_filename(__name__, "images")
-IMAGE_PATH = os.path.join(IMAGE_DIR, "core.png")
+SEARCH_PATH = os.fspath(files("pyface") / "images")
+IMAGE_DIR = files("pyface.tests") / "images"
+IMAGE_PATH = os.fspath(IMAGE_DIR / "core.png")
```

Comment:
Nit: The `IMAGE_DIR` is not converted, which looks inconsistent. Turns out it is only used once here, so we could also just remove `IMAGE_DIR` and have `IMAGE_PATH = os.fspath(files("pyface.tests") / "images" / "core.png")` instead.

Reply:
done.